schemas/2014-04-01-preview/deploymentTemplate.json

```diff
@@ -538,6 +538,11 @@
                   { "$ref": "https://schema.management.azure.com/schemas/2017-11-01/Microsoft.Network.json#/resourceDefinitions/networkSecurityGroups" },
                   { "$ref": "https://schema.management.azure.com/schemas/2017-11-01/Microsoft.Network.json#/resourceDefinitions/networkInterfaces" },
                   { "$ref": "https://schema.management.azure.com/schemas/2017-11-01/Microsoft.Network.json#/resourceDefinitions/routeTables" },
+                  { "$ref": "https://schema.management.azure.com/schemas/2017-03-30/Microsoft.Compute.json#/resourceDefinitions/images" },
```

Comment:
>2017-03-30 [](start = 73, length = 10)

is this correct? should it be "2017-12-01"?

Reply:
Fixed